cmd/checkmodels/main.go

```diff
@@ -24,7 +24,7 @@ var types = []Type{
 	Type{typelabel.Count, typelen.Count},
 	Type{typelabel.Enum16, typelen.Enum16},
 	Type{typelabel.Enum32, typelen.Enum32},
-	Type{typelabel.Eui48, typelen.Eui48},
+	Type{typelabel.EUI48, typelen.EUI48},
```

Comment:
Stylistic complaint, I pref acronyms to maintain camel case (e.g. `Ipv6` and `Uint` below)